staging/src/k8s.io/api/autoscaling/v2beta1/types.go

```diff
@@ -78,7 +78,8 @@ var (
 // MetricSpec specifies how to scale based on a single metric
 // (only `type` and one other matching field should be set at once).
 type MetricSpec struct {
-	// type is the type of metric source.  It should match one of the fields below.
+	// Type is the type of metric source.  It should be one of "Object",
```

Comment:
Keep type lowercase to match the JSON API name

Reply:
Done.